jobs/cjd_sample/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.samples</groupId>
+  <artifactId>cloud-job-discovery-samples</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+
+  <properties>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.source>1.8</maven.compiler.source>
+  </properties>
+
```

Comment:
This project should use the `shared-configuration` parent project, for reference check other `pom.xml`'s in this repo. After adding that, please run the `checkstyle:checkstyle` maven task to make sure everything's kosher there as well.

Also, new pom.xml's that are created should specify a minimum version of Maven, as that's a new requirement. See [here](http://shortn/_iZ6JHNEyXc) for an example.

Reply:
Sorry. I am not too familiar with Maven, may you specify the details? The content of  "properties" are copied from the pom file under the pubsub sample folder.

And I have added 'prerequisites' as you recommended.